bin/hokusai

```diff
@@ -72,14 +72,42 @@ def check():
   hokusai.check()
 
 @cli.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--skip-build', type=click.BOOL, is_flag=True, help='Do not build the project while launching the stack')
+@click.argument('action', type=click.STRING)
+@click.option('-s', '--skip-build', type=click.BOOL, is_flag=True, help='Do not build the project while launching the stack')
+@click.option('-f', '--follow', type=click.BOOL, is_flag=True, help="Follow stack output when running 'start' or 'logs'")
```

Comment:
Hmm although `follow` is used in docker-compose as well as kubernetes - also we have this option in `hokusai logs` so it may cause confusion changing it up.  I think it's bes tto stick with the naming applied by the tools Hokusai is wrapping